int32 size
hebiros/EntryMsg[] entries
